addons/mail/models/mail_channel.py

```diff
@@ -693,18 +707,14 @@ def channel_fold(self, uuid, state=None):
                 vals['is_minimized'] = is_minimized
             if vals:
                 session_state.write(vals)
-            self.env['bus.bus'].sendone((self._cr.dbname, 'res.partner', self.env.user.partner_id.id), session_state.channel_id.channel_info()[0])
-
-    @api.model
-    def channel_minimize(self, uuid, minimized=True):
-        values = {
-            'fold_state': minimized and 'open' or 'closed',
-            'is_minimized': minimized
-        }
-        domain = [('partner_id', '=', self.env.user.partner_id.id), ('channel_id.uuid', '=', uuid)]
-        channel_partners = self.env['mail.channel.partner'].search(domain, limit=1)
-        channel_partners.write(values)
-        self.env['bus.bus'].sendone((self._cr.dbname, 'res.partner', self.env.user.partner_id.id), channel_partners.channel_id.channel_info()[0])
+            self.env['bus.bus'].sendone(
+                self.env.user.partner_id,
+                'mail.fold_chat_window',
+                {
+                    'id': session_state.channel_id.channel_info()[0]['id'],
+                    'state': state,
+                }
+            )
```

Comment:
payload change: I think we should split fold and unfold in 2 different notifications (if not, the type should be along the lines of "fold state changed" instead of specifically "fold").